ui/pages/Search/components/VariantSearchForm.jsx

```diff
@@ -33,3 +34,11 @@ const mapStateToProps = (state, ownProps) => ({
 })
 
 export default connect(mapStateToProps)(VariantSearchForm)
+
+const mapNoEditProjectStateToProps = (state, ownProps) => ({
+  noEditProjects: true,
```

Comment:
The `noEditProjects` is always `true`. Are there any use cases that the `noEditProjects` are false?

Reply:
It is always true for the `NoEditProjectsVariantSearchForm` and always false for the regular `VariantSearchForm`